pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -263,6 +267,44 @@ func (s *Server) initConfigSources(args *PilotArgs) (err error) {
 			}
 			s.ConfigStores = append(s.ConfigStores, configController)
 			log.Infof("Started XDS configSource %s", configSource.Address)
+		case DelataXDS:
+			store := memory.Make(collections.Pilot)
+			// TODO: enable namespace filter for memory controller
+			configController := memory.NewController(store)
+
+			delta := adsc.NewDelta(srcAddress.Host,
+				&adsc.DeltaADSConfig{
+					Config: adsc.Config{
```

Comment:
nit: would be nice to share adsc.Config with the non-delta case

Reply:
Indeed the XDS and DeltaXds branches in pr actually have a lot of duplicated code, because it's not clear what method is used to determine whether delta-xds is activated, so  all of them are temporarily split into two cases.